packages/components/src/gradient-picker/stories/index.tsx

```diff
@@ -6,18 +10,23 @@ import { useState } from '@wordpress/element';
 /**
  * Internal dependencies
  */
-import GradientPicker from '../';
+import GradientPicker from '..';
 
-export default {
+const meta: ComponentMeta< typeof GradientPicker > = {
 	title: 'Components/GradientPicker',
 	component: GradientPicker,
+	parameters: {
+		controls: { expanded: true },
+		docs: { source: { state: 'open' } },
+	},
```

Comment:
Here's few minor improvements that we could make:

- the configuration for the `boolean` controls specified under the `argTypes` property can probably be deleted, since Storybook should now be able to infer it via TypeScript
- as a personal preference, I ted to prefer adding global `actions` settings under the `parameters` property, than the individual action control (ie. I would delete `onChange: { action: 'onChange' },` under the `argTypes` property, and add `actions: { argTypesRegex: '^on.*' },` under the `parameters` property)
- we should disable controls for the `value` prop, since the Storybook example is controlled